routes/web.php

```diff
@@ -1,6 +1,7 @@
 <?php
```

Comment:
I'll review the routes/web.php file.

The code looks good, but there's one potential issue:

In Laravel, it's recommended to use the `Route::group` method to organize routes into logical groups, rather than having them all at the top level of the file.

Example:
```php
Route::group(['middleware' => 'auth'], function () {
    Route::get('/dashboard', 'DashboardController@index');
});
```
This way, you can keep related routes together and apply middleware to specific groups of routes.

No other issues are found.